Doppler.Jobs.Server/Startup.cs

```diff
@@ -71,7 +71,7 @@ public void ConfigureServices(IServiceCollection services)
                 sp.GetService<IHttpClientFactory>(), 
                 httpClientPolicies,
                 dopplerCurrencySettings,
-                sp.GetService<ILogger<BaseExternalService>>(),
+                sp.GetService<ILogger<DopplerCurrencyService>>(),
```

Comment:
Is it not possible simply declaring the implementation type like:

```csharp
            services.AddTransient<DopplerCurrencyService>()
```

Then, when you want to implemenit, the dependencies will be resolved directly.

Reply:
Yes done https://github.com/MakingSense/doppler-jobs/pull/2/commits/932b523fac350bbf9be7fa7e2302d4b609a1650b